composer.json

```diff
@@ -41,7 +41,8 @@
         "phpunit/phpunit": "3.7.*",
         "mikey179/vfsStream": "~1.2.0",
         "league/flysystem": "~1.0",
-        "mongodb/mongodb": "~1.1.4"
+        "mongodb/mongodb": "~1.1.4",
+        "google/cloud-storage": "dev-master"
```

Comment:
`google/cloud-storage` v1.0 was released a month ago. Do you think your work is compatible with this branch? Would be great to depend on a stable branch rather than `dev-master` ;)